Index.html

```diff
@@ -27,16 +17,18 @@
         <a href="#" class="logo">
             <img src="./images/logo.svg" alt="Logo" />
         </a>
-        <nav class="navbar">
+       
             <a href="#" class="menu">
-                <img src="./images/Union.svg" alt="Menu" />
+                <img src="./images/Union.svg"  alt="Menu" />
             </a>
-            <ul class=nav-menu>
-                <li><a href="#" class="nav-portfolio">Portfolio</a></li>
-                <li><a href="#" class="nav-about">About</a></li>
-                <li><a href="#" class="nav-connect">Contact</a></li>
-            </ul>
-        </nav>
+            <nav class="navbar">
+                <ul class="nav-menu">
```

Comment:
- [ OPTIONAL ] Please prevent the nav menu from scrolling with it is toggled, this will make its use more meaningful and professional for the user.